components/collection/unlockable/UnlockableContainer.vue

```diff
@@ -237,12 +243,18 @@ const refetchData = async () => {
 }
 
 useSubscriptionGraphql({
-  query: `collectionEntityById(id: "${collectionId}") {
+  query: `collectionEntityById(id: "${collectionId.value}") {
     nftCount
   }`,
   onChange: refetchData,
 })
 
+watch(accountId, () => {
+  tryAgain({
```

Comment:
shouldn't it be async?

Reply:
It is not needed because this function blocks no other task.